bento-sample-app/src/main/java/com/yelp/android/bentosampleapp/ListViewActivity.kt

```diff
@@ -1,6 +1,7 @@
 package com.yelp.android.bentosampleapp
```

Comment:
Be aware that the ListViewActivity was created to verify that list views display stuff properly. There is no link between the ListViewActivity and the ListComponent. Bento is compatible with both ListViews and RecyclerViews, ViewPagers...